### PR TITLE
Append ApplyFunction to math functions

### PR DESCRIPTION
**Pull request by ronkok**
Append a `<mo>&ApplyFunction;</mo>` to the MathML of math functions such as `\sin` or `\tan`.

Ref:  https://www.w3.org/TR/REC-MathML/chap3_2.html#sec3.2.4

### CODE REVIEW

File: test/__snapshots__/mathml-spec.js.snap [hunk 1]
```diff
@@ -27,6 +27,9 @@ exports[`A MathML builder should generate the right types of nodes 1`] = `
       <mi>
         sin
       </mi>
+      <mo>
+        &amp;ApplyFunction;
```
Comment (edemaine) [inline test/__snapshots__/mathml-spec.js.snap]: I think this should be `&ApplyFunction;`, right? `makeText` above must be escaping the ampersand in a way we don't want...

Comment (ronkok) [reply test/__snapshots__/mathml-spec.js.snap]: I missed that. Thanks for the comment.

Comment (edemaine) [reply test/__snapshots__/mathml-spec.js.snap]: Is there a reference to show that `&ApplyFunction;` is equivalent to Unicode Character 'FUNCTION APPLICATION' (U+2061) in MathML?  It seems to be, but I couldn't find one...

Comment (ronkok) [reply test/__snapshots__/mathml-spec.js.snap]: This [cheat sheet](http://www.html-5.com/cheat-sheet/html-character-codes.html) gives the equivalence. You can also type in `\sin x`  into the LaTeX box at [iMathEQ](http://www.imatheq.com/imatheq/com/imatheq/math-equation-editor-latex-mathml.html) and ask it to give you their version of the equivalent MathML.

Comment (ronkok) [reply test/__snapshots__/mathml-spec.js.snap]: [This page](https://www.w3.org/TR/xml-entity-names/bycodes.html) is from a better source.

**Comment by edemaine** [comment]
LGTM! Thanks for the pointers.

**Comment by ronkok** [comment]
@kevinbarabash, I think I will shut this PR down and re-submit after PR #871 lands. That will be easier than trying to coordinate the two PRs.

**Comment by kevinbarabash** [comment]
@ronkok thanks.  I'll see what I can do to expedite that review.

**Pull request closed without merging**